feat(M&B): Use cases (#4857)
Kong/developer.konghq.com: app/metering-and-billing/entitlements.md

Reviewer: You'll want to replace the Entitlements tile link on the landing page: https://developer.konghq.com/metering-and-billing/#product-catalog

      <br><br>
      In cases where you don't need to set up usage limits or configure customer-level settings,
      you can use boolean entitlements. 
      These are simple `true` or `false` access grants to a feature.
{% endtable %}

### Metered entitlements

Metered entitlements control access to features where you want to impose usage limits. 
They leverage the usage information collected by {{site.metering_and_billing}} to enable real-time usage enforcement.

#### Usage period

Metered entitlements require a usage period setting that defines the interval over which usage is calculated. This is typically a day, week, or month. In most cases, this should match your customers' billing cycles. 

The start of each period is marked by a reset: at reset, a new marker is set from which usage is queried and aggregated.
{{site.metering_and_billing}} automatically executes the reset when a new usage period starts, based on the entitlement configuration.

If you want to maintain a continuous running balance where overage carries forward rather than being forgiven at the end of each period, set the **Preserve Overage** field on the entitlement.

> When toggled **on**, overage accumulated in the previous period is deducted from the starting balance of the new period.

Reviewer: ```suggestion
When this is enabled, overage accumulated in the previous period is deducted from the starting balance of the new period.
```
